content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -196,17 +197,33 @@ parameters:
 allowVolumeExpansion: true
 ```
 
-Once both feature gate and the aforementioned admission plug-in are turned on, a user can request larger volume for their `PersistentVolumeClaim`
-by simply editing the claim and requesting a larger size.  This in turn will trigger expansion of the volume that is backing the underlying `PersistentVolume`.
+To request a larger volume for a PVC, edit the PVC's configuration and request a larger
```

Comment:
I am not sure I would use wording "edit the PVC's configuration" to refer editing the pvc's yaml - I was thinking something like "edit PVC object" would be better?